core/src/price_finding/price_finder_interface.rs

```diff
@@ -138,7 +138,7 @@ pub fn execute_private_solver(
         .arg(format!("{}{}", "/app/", input_file.to_owned()))
         .arg(format!("--outputDir={}{}", "/app/", result_folder))
         .arg("--logging=WARNING")
-        .args(&["--solverTimeLimit", &time_limit])
+        .args(&["--timeLimit", &time_limit])
```

Comment:
uber-nano-nit: for consistency, we should probably change this to be formatted like the other arguments
```suggestion
        .arg(format!("--timeLimit={}", time_limit))
```